go.mod

```diff
@@ -4,11 +4,10 @@ go 1.16
 
 require (
 	cloud.google.com/go v0.82.0
-	cloud.google.com/go/logging v1.1.2
 	cloud.google.com/go/storage v1.12.0
 	github.com/GoogleCloudPlatform/testgrid v0.0.38
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/cenkalti/backoff/v4 v4.1.1
+	github.com/cenkalti/backoff/v4 v4.1.1 // indirect
```

Comment:
nit: remove this and run `go mod tidy`

Reply:
The bots are too fast!
go.mod fix here: https://github.com/kubernetes/release/pull/2102